### ISSUE TITLE
htop install issue on MacOSX

### ISSUE THREAD

**Issue by palmerj** [linz/ansible-demo#1]
When bringing up all of the VMs I get an error for the task that installs htop:

failed: [db] => (item=[u'htop', u'mc', u'tmux']) => {"failed": true, "item": ["htop", "mc", "tmux"], "msg": "No package matching 'htop' is available"}

**Comment by imincik** [comment]
Hi @palmerj,
I have no immediate idea what can cause such problem. It can't be related
to OSX. I will investigate ASAP.

Would you please paste complete output from 'vagrant up'  command?

**Comment by palmerj** [comment]
Output from vagrant up:

https://gist.github.com/palmerj/c5f8659585edba522e5160bc756cce1c

**Comment by imincik** [comment]
Thanks,
I have some ideas now. I will investigate more tomorrow.

**Comment by palmerj** [comment]
Great thanks.

**Comment by imincik** [comment]
@palmerj,
I have no access to Mac.
Can you please try to apply following small patch:

```
--- a/provision/roles/basic/tasks/main.yml
+++ b/provision/roles/basic/tasks/main.yml
@@ -48,7 +48,7 @@
 - name: Update Apt cache
   apt:
     update_cache: yes
-    cache_valid_time: 3600
+#    cache_valid_time: 3600
   environment: "{{ proxy_env }}"
   become: yes
```

and run _vagrant up_ with _--no-parallel_ option ?

```
$ vagrant up --no-parallel
```

Thanks.

**Comment by palmerj** [comment]
Worked! Thanks. 

I also had to install curl package for the load balancer test task. Ansible also noted to consider using get_url module rather than running curl.

**Comment by imincik** [comment]
curl added in 5923bf065609c10e31280275e1f5b91b1c6e895a

**Comment by palmerj** [comment]
Fixed in 3278fe2
